app/main.py

```diff
@@ -1,2 +1,66 @@
+from typing import Any
+
+
 class Dictionary:
-    pass
+    def __init__(self) -> None:
+        self.length: int = 0
+        self.capacity: int = 8
+        self.hash_table: list[tuple | None] = [None] * self.capacity
```

Comment:
The use of 'tuple | None' for type hinting in the list is incorrect for Python versions below 3.10. Consider using 'Optional[Tuple]' from the 'typing' module instead.